### DISCUSSION TITLE
Add Custom Breakpoints to Editor Header

### DISCUSSION THREAD

**Discussion by tomektomczuk** [WordPress/gutenberg#45568] [Q&A]
So at the moment we have 3 base preview Mobile, Tablet and Desktop. Is there a way to include custom preview options and add them to the centre of editor header like in the attached image?
<img width="1017" alt="Screenshot 2022-11-05 at 19 47 52" src="https://user-images.githubusercontent.com/40096666/200136122-e6f46e75-df1d-41cf-b092-c8e83e621d11.png">

**Comment by skorasaurus** [comment]
Not yet, this is currently being tracked in https://github.com/WordPress/gutenberg/issues/21235
